src/meta_type.rs

```diff
@@ -111,3 +114,25 @@ impl MetaType {
         self == &MetaType::new::<crate::impls::PhantomIdentity>()
     }
 }
+
+#[cfg(all(feature = "std", feature = "schema"))]
+impl schemars::JsonSchema for MetaType {
```

Comment:
What do we need this for?

Reply:
Indeed, I was hoping that the `derive(JsonSchema)` was clever enough to add associated trait bounds e.g. `where T::Type: JsonSchema. Sadly not though so we do need this.